readthedocs/builds/models.py

```diff
@@ -151,6 +152,9 @@ def commit_name(self):
         log.debug('TODO: Raise an exception here. Testing what cases it happens')
         return self.identifier
 
+    def get_url(self):
+        return resolve_path(self.project, version_slug=self.slug)
```

Comment:
Maybe `get_docs_url` or similar is a better name for this method.

Also, there is a `get_subdomain_url` method for this object. It may worth to take a look at what it does because we probably need something similar here (use the `project.get_docs_url`).